pkg/cmd/server/kubernetes/master/master_config.go

```diff
@@ -507,6 +505,20 @@ func buildKubeApiserverConfig(
 		return originLongRunningRequestRE.MatchString(r.URL.Path) || kubeLongRunningFunc(r, requestInfo)
 	}
 
+	if apiserverOptions.Etcd.EnableWatchCache {
+		glog.V(2).Infof("Initializing cache sizes based on %dMB limit", apiserverOptions.GenericServerRunOptions.TargetRAMMB)
+		sizes := cachesize.NewHeuristicWatchCacheSizes(apiserverOptions.GenericServerRunOptions.TargetRAMMB)
```

Comment:
do we set this target RAMMB to anything by default?

Reply:
> do we set this target RAMMB to anything by default?

I'm not seeing where we write a default here, which would set all the heuristic ones to zero by default, right?

Reply:
There's a "min" function on the heuristic so we always get something even at 0